application/helpers/admin/import_helper.php

```
@@ -2485,7 +2492,7 @@ function XMLImportResponses($sFullFilePath, $iSurveyID, $aFieldReMap = array())
                                 $results['responses']++;
                             } else {
                                 if (!empty($aInsertData['id'])) {
-                                    $results['warnings'][] = CHtml::errorSummary($response, "<div>" . sprintf(gT("Failed to save response data : response id %s"), 'unescaped'). "</div>");
+                                    $results['warnings'][] = CHtml::errorSummary($response, "<div>" . sprintf(gT("Failed to save response data : response id %s"), 'unescaped') . "</div>");
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'CHtml' in method 'XMLImportResponses'. (undefined)

(StaticAccess)